nilearn/reporting/tests/test_reporting.py

```diff
@@ -142,10 +148,7 @@ def test_get_clusters_table(tmp_path):
     data_extra_dim = data[..., np.newaxis]
     stat_img_extra_dim = nib.Nifti1Image(data_extra_dim, np.eye(4))
     cluster_table = get_clusters_table(
-        stat_img_extra_dim,
-        4,
-        0,
-        two_sided=True
+        stat_img_extra_dim, 4, 0, two_sided=True
```

Comment:
Either everything fits on one line or you have to put one argument per line.

Reply:
Thanks for catching this. It looks like the lack of a trailing comma caused `black` to reformat that into a single line